modules/ROOT/pages/_partials/include-traffic-inspection.adoc

```diff
@@ -0,0 +1,90 @@
+
+
+//
+//tag::traffic-inspection-overview[]
+Intercept and inspect all HTTPS traffic through a proxy server that resides between your on-premises Mule apps and Anypoint Monitoring. To enable traffic inspection for your xref:runtime-manager::deployment-strategies.adoc#hybrid-deployments[hybrid deployments], you must install the latest Anypoint Monitoring agent.
```

Comment:
Do we need to mention the version 3.x?

Reply:
@bin-shi-mulesoft 3.9 is in end of life support for on-prem.

Reply:
What I mean is the Anypoint Monitoring agent 3.x. I think we may not need to mention as it is not released.